chimp.md

```diff
@@ -0,0 +1,213 @@
+- **Payment Details:**  0x (Add later)
+
+
+## Project Overview :page_facing_up:
+
+
+### Overview
+
+Chimp Exchange is the first private decentralized exchange (DEX) offering complete anonymity for traders by leveraging end-to-end encryption (E2EE) via the Ten Protocol. With privacy becoming a critical concern in the DeFi and Web3 ecosystem, Chimp Exchange fills the gap by providing a fully encrypted trading platform, ensuring that trades, asset movements, and identities are kept private and secure. This innovative DEX prevents tracking of trades and assets, thus protecting whales, traders, and privacy-conscious users from the risks associated with public visibility on blockchain networks.
+
+Chimp Exchange aims to be the bridge to a fully private trading future, offering users the ability to trade without fear of tracking, identity disclosure, or liquidation hunting. The platform is currently live on Testnet with Cross-Chain capabilities and boasts a rapidly growing user base and community.
+
+### Project Details
+
+- Key Features:
+
+	•	Encrypted Trading: Utilizes end-to-end encryption to ensure privacy in trading, making it impossible to trace asset flows.
+	•	Hidden Liquidity Pools: Maintains privacy for liquidity providers by preventing visibility and tracking.
+	•	Private Fundraising Launchpad: Enables anonymous fundraising for projects while maintaining a high level of security.
+	•	Cross-Chain Compatibility: Initially deployed on Linea and scaling through Ten Protocol for more robust cross-chain trading solutions.
+	•	Protection Against Asset and Trade Tracking: Safeguards users from tracking, doxxing, and liquidity hunting, ensuring a secure trading experience.
+
+- Current Achievement and Traction:
+
+	•	Testnet Product: Already deployed on Ten Protocol, with a successful demo showcasing the first fully encrypted DEX.
+	•	Mainnet on Linea: The cross-chain DEX on Linea Mainnet has gathered over 10,000 users and is experiencing organic growth through an ambassador program with over 180 ambassadors.
+	•	Community Growth: Strong community engagement with 10,000+ active members on Discord and Twitter.
+	•	Upcoming Testnet Launch: Launch on 22nd May showcasing the next phase of the private DEX, allowing for further testing and engagement.
+
+### Ecosystem Fit
+
+Chimp Exchange fits into the Polkadot/Substrate/Kusama ecosystem by addressing the critical gap in privacy and anonymity in decentralized finance (DeFi). While Polkadot’s interoperability allows seamless cross-chain functionality, Chimp Exchange introduces a private decentralized exchange (DEX) that focuses on enabling anonymous and encrypted trading. By leveraging end-to-end encryption (E2EE) via Ten Protocol, Chimp Exchange provides a secure, private platform for users to trade digital assets across various parachains. This ensures that sensitive information, such as trade details and wallet balances, remains hidden from third-party tracking.
```

Comment:
To my understanding the Manta parachain is doing something with similar goals. Have you looked into Manta previously? I would like to know why you think your solution has an edge on Manta and what this benefit is exactly.